setup.py

```diff
@@ -33,17 +33,17 @@ def read(*rnames):
         'jinja2',
         'packaging',
         'pygments',
+        'pyyaml',
         'reportlab',
         'smartypants',
-        'pyyaml',
     ],
     extras_require={
-        'tests': ['pyPdf2', 'pymupdf'],
-        'sphinx': ['sphinx'],
-        'svgsupport': ['svglib'],
         'aafiguresupport': ['aafigure>=0.4'],
         'mathsupport': ['matplotlib'],
         'rawhtmlsupport': ['xhtml2pdf'],
+        'sphinx': ['sphinx'],
+        'svgsupport': ['svglib'],
+        'tests': ['pyPdf2', 'pymupdf', 'pytest'],
```

Comment:
> * why do we require pypdf2 for tests, but not overall (it's brought in by xhtml2pdf in the requirements files)?

Because we always have?! 😆

Looking into it, it was a requirement for the very first test code that was removed in [November 2009](https://github.com/rst2pdf/rst2pdf/commit/93dcc79531b4d0a18b954eb283e271e93ceb4cc0)

Reply:
Removed in 93e1da3e8d7a10ba273d6317641dc8a9de184f3b.

Reply:
> * should plantuml be here as an optional extra?

Yes!  Should have been there all along really.

Added in afe251092d06dc67185b06cc67a25a4e34994691 and also added info into README on how to install extras using pip.